Yosemite/Yosemite/Model/Model.swift

```diff
@@ -111,6 +111,7 @@ public typealias CardReaderSoftwareUpdate = Hardware.CardReaderSoftwareUpdate
 public typealias CardReaderServiceDiscoveryStatus = Hardware.CardReaderServiceDiscoveryStatus
 public typealias CardReaderServiceError = Hardware.CardReaderServiceError
 public typealias PaymentParameters = Hardware.PaymentIntentParameters
+public typealias PaymentIntent = Hardware.PaymentIntent
```

Comment:
I don't love this. I don't like the idea of exposing the PaymentIntent. On the other hand, even though at this point we only need the payment intent identifier outside of Yosemite, I have the intuition that it won't hurt to expose it anyway for the near future. 

I'm happy to drop a follow up PR to do whatever we decide to do